sw/device/lib/testing/test_framework/BUILD

```diff
@@ -44,3 +44,79 @@ cc_library(
         "//sw/device/lib/runtime:print",
     ],
 )
+
+# TODO(lowRISC/opentitan#7972): clean up check.h after completing the
+# migration to bazel.
+cc_library(
+    name = "check",
+    hdrs = [
+        "test_status.h",
+        "//sw/device/lib/testing:check",
+    ],
+    visibility = ["//visibility:private"],
+)
+
+cc_library(
+    name = "freertos_config",
+    hdrs = [
+        "FreeRTOSConfig.h",
+    ],
+    includes = ["."],
```

Comment:
Can we have a comment explainin why this is necessary?

Reply:
Done.